src/uikit.cs

```diff
@@ -15405,6 +15435,24 @@ partial interface UIInputViewController : UITextInputDelegate {
 		void HandleInputModeList (UIView fromView, UIEvent withEvent);
 	}
 
+	[NoWatch, NoTV, iOS (11,0)]
+	[Protocol, Model]
```

Comment:
This protocol is supposed to be implemented by existing types, so `[Model]` (and `[BaseType]`) should be dropped.